### PR TITLE
Enable twig profiler if available

### PR DESCRIPTION
**Pull request by glaubinix**
Enables to twig profiler which was introduced with twig 1.18.

The profiler can also be used with the 1.0 branch and silex 1.3. but I wasnt really sure which profiler branch will be used for silex 1.3.

### CODE REVIEW

**Comment by lschricke** [comment]
This would be a great addition!

File: WebProfilerServiceProvider.php [hunk 1]
```diff
@@ -102,6 +105,18 @@ public function register(Container $app)
             });
         }
 
+        if (class_exists('\Symfony\Bridge\Twig\Extension\ProfilerExtension')) {
```
Comment (fabpot) [inline WebProfilerServiceProvider.php]: Can you remove the leading `\` as it's not needed.

File: WebProfilerServiceProvider.php [hunk 2]
```diff
@@ -102,6 +105,18 @@ public function register(Container $app)
...
+            $app->extend('data_collectors', function ($collectors, $app) {
+                $collectors['twig'] = function ($app) { return new TwigDataCollector($app['twig.profiler.profile']); };
```
Comment (fabpot) [inline WebProfilerServiceProvider.php]: Can you move the `return new ...` line onto a new line?

**Comment by glaubinix** [comment]
@fabpot moved return and removed leading ````\````

**Comment by fabpot** [comment]
Looks good. I've just realized that this PR is done on master but I would prefer to merge it in the 1.0 branch. Can you submit a PR for the 1.0 branch instead? Thanks.

**Comment by glaubinix** [comment]
@fabpot Done. Closing this in favour of #55. I cant switch target branch of a PR.

**Pull request closed without merging**